install.sh

```diff
@@ -167,7 +166,6 @@ while getopts "hsinaf" opt; do
 		"f") overwrite_backup=true ;;
 		"?")
 			_bash-it_show_usage >&2
-			exit 1
```

Comment:
Just to be clear, my recommendation was the OTHER way - remove the `exit 0` from `_bash-it_show_usage` ...
This is so that the code can vary the exit code (ie `1` here, but `0` above) based on context ...

Reply:
Done 👍🏻